client/stylesheets/_responsive.scss

```diff
@@ -289,6 +276,14 @@
     h4 {
         font-size: 1rem;
     }
+
+    .profileCardInfo li:first-child {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardInfo` should be written in lowercase with hyphens